modules/persistent-storage-csi-smb-cifs-create-pv.adoc

```diff
@@ -0,0 +1,192 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-smb-cifs.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="persistent-storage-csi-smb-cifs-create-pv_{context}"]
+= Creating PVs/PVCs with SMB shares
+
+To create a persistent volume (PV) and persistent volume claim (PVC) to consume Server Message Block protocol (SMB) shares:
+
+.Prerequisites
+* Access to the {product-title} web console.
+* {FeatureName} CSI Driver Operator and driver installed.
+* Storage class for CIFS/SMB already created (see _Creating a storage class for CIFS/SMB_).
+
+.Procedure
+
+. Create a PV:
++
+Run the following command with the following example YAML file:
++
+[source,cli]
```

Comment:
Per the [doc guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#source-tags-for-terminal-commands-and-output) I think the source type should be terminal:
```suggestion
[source,terminal]
```